distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Split_Tokenize.enso

```diff
@@ -0,0 +1,246 @@
+
+    0.up_to num_input_rows . map i->
+        input_value = input_storage.getItemBoxed i
+        output_values = function input_value
+        # Append each value.
+        output_values.map v-> output_column_builder.append v
+        # Append n copies of the input row position, n = # of output values.
+        repeat output_values.length <| order_mask_positions.append i
+
+    # Build the output column
+    output_storage = output_column_builder.seal
+    output_column = Column.from_storage input_column_name output_storage
+
+    # Build the order mask.
+    order_mask = OrderMask.new (order_mask_positions.to_vector)
+
+    # Build the other columns, and include the output_column while doing it.
+    new_columns = table.columns.map column->
+        case column.name == input_column_name of
+            True ->
+                # Replace the input column with the output column.
+                output_column
+            False ->
+                # Build a new column from the old one with the mask
+                old_storage = column.java_column.getStorage
+                new_storage = old_storage.applyMask order_mask
+                Column.from_storage column.name new_storage
+
+    Table.new new_columns
+
+## PRIVATE
+
+   Map a multi-valued function over a column and return the results as set of
+   output columns.
+
+   Returns a Pair of a Vector of Columns and a Vector of problems.
+
+   Arguments:
+   - input_column: The column to transform.
+   - function: A function that transforms a single element of `input_column`
+     to multiple values.
+   - column_count: The number of columns to split to.
+     If `Auto` then columns will be added to fit all data.
+     If the data exceeds the `column_count`, a `Column_Count_Exceeded` error
+     will follow the `on_problems` behavior.
+   - on_problems: Specifies the behavior when a problem occurs.
+map_columns_to_multiple : Column -> (Any -> Vector Any) -> Auto | Integer -> Problem_Builder -> Vector Column
+map_columns_to_multiple input_column function column_count problem_builder =
+    num_rows = input_column.length
+    input_storage = input_column.java_column.getStorage
+
+    builders = case column_count of
+        Auto ->
+            builders = Vector.new_builder
+
+            0.up_to num_rows . map i->
+                input_value = input_storage.getItemBoxed i
+                output_values = function input_value
+
+                # Add more builders if necessary to accommodate `output_values`.
+                if output_values.length > builders.length then
+                    num_builders_needed = output_values.length - builders.length
+                    repeat num_builders_needed <|
+                        builder = make_string_builder num_rows
+
+                        # Pad the new builder with nulls
+                        num_nulls_needed = i
+                        builder.appendNulls num_nulls_needed
+
+                        builders.append <| builder
```

Comment:
```suggestion
                        builders.append builder
```

I don't see why we'd need `<|` here at all?

Reply:
Used to be more after the <| :)